packages/apfelxx/download.sh

```diff
@@ -0,0 +1,4 @@
+URL=$1
+TARBALL=$2
+
+wget $URL -O $TARBALL
```

Comment:
```suggestion

git clone -b resummation-scales $URL
```

Reply:
If you use the tarball URL, you can keep this exactly as it is :)